app/benchmarking.py

```diff
@@ -56,54 +61,182 @@ def run(circuit, backend, token, shots, benchmark_id, original_depth, original_w
-            # create randomized circuits and transpile them for both backends
-            qx = random_circuit(num_qubits=num_of_qubits, depth=i, measure=True)
-            original_number_of_multi_qubit_gates = qx.num_nonlocal_gates()
-            qcircuit_sim = transpile(qx, backend=backend_sim, optimization_level=3)
-            qcircuit_real = transpile(qx, backend=backend_real, optimization_level=3)
-            # ensure that the width of the circuit is correctly saved in the db
-            remove_final_meas = RemoveFinalMeasurements()
-            active_qubits_real = [
-                qubit for qubit in qcircuit_real.qubits if
-                qubit not in remove_final_meas.run(circuit_to_dag(qcircuit_real)).idle_wires()
-            ]
-            transpiled_depth_sim = qcircuit_sim.depth()
-            transpiled_width_sim = qcircuit_sim.num_qubits
-            transpiled_number_of_multi_qubit_gates_sim = qcircuit_sim.num_nonlocal_gates()
-            transpiled_depth_real = qcircuit_real.depth()
-            transpiled_width_real = len(active_qubits_real)
-            transpiled_number_of_multi_qubit_gates_real = qcircuit_real.num_nonlocal_gates()
-
-            location_sim = run(circuit=qcircuit_sim, backend=sim_name, token=token, shots=shots,
-                               benchmark_id=benchmark_id,
-                               original_depth=i, original_width=num_of_qubits,
-                               original_number_of_multi_qubit_gates=original_number_of_multi_qubit_gates,
-                               transpiled_depth=transpiled_depth_sim, transpiled_width=transpiled_width_sim,
-                               transpiled_number_of_multi_qubit_gates=transpiled_number_of_multi_qubit_gates_sim)
-
-            location_real = run(circuit=qcircuit_real, backend=qpu_name, token=token, shots=shots,
-                                benchmark_id=benchmark_id,
-                                original_depth=i, original_width=num_of_qubits,
-                                original_number_of_multi_qubit_gates=original_number_of_multi_qubit_gates,
-                                transpiled_depth=transpiled_depth_real, transpiled_width=transpiled_width_real,
-                                transpiled_number_of_multi_qubit_gates=transpiled_number_of_multi_qubit_gates_real)
-            location_benchmark = '/qiskit-service/api/v1.0/benchmarks/' + str(benchmark_id)
-            locations.append({'result-simulator': str(location_sim),
-                              'result-real-backend': str(location_real),
-                              'result-benchmark': str(location_benchmark)})
+    if clifford:
+        provider = IBMQ.get_provider(group='open')
+        backend_real = provider.get_backend(qpu_name)
+        backend_sim = provider.get_backend(sim_name)
+        nseeds = num_of_circuits
+
+        # random rb_pattern with random number of multi-qubit-gates
+        qubit_seq = [j for j in range(num_of_qubits)]
```

Comment:
comment the steps, e.g. that the following steps here are to create randomized Clifford gate circuits

Reply:
so set the comment below